hpi-swt2/lecture-portal
# Feature/#74 assign questions to lecture

# Resolves #74 

This PR changes the feature of questions to be shown in the lecture UI (only)

Acceptance Criteria:
- [x] Students and lecturers have access to this feature
- [x] See mockup
- [x] Only questions assigned to the current lecture are shown
- [x] Every question is assigned to the lecture in which it was asked
- [x] Questions are shown in the "Updates" section of the dashboard
- [x] Questions can be resolved by lecturers by clicking the question in the update section of the dashboard 


Testing steps:
- as a student, join a lecture and go to "Questions" (/questions) and see or ask questions (of this lecture only)
- as a lecturer, create a lecture and go to "Questions" (/questions) and see questions (of this lecture only)
- as either lecturer or student, go to a lecture's dashboard and see incoming questions in the update section
- as a lecturer, resolve questions in the dashboard's update section by clicking on them

Steps needed for migration:
- rails db:migrate

---

Definition Of Done:
